src/renderer/windows/settings/Settings.vue

```diff
@@ -396,10 +407,23 @@ window.ytmd.handleUpdateDownloaded(() => {
               <p class="description">
                 User is Authenticated:
                 <span v-if="lastFMSessionKey" style="color: #4caf50">Yes</span>
+                <button v-if="lastFMSessionKey" class="lastfm-logout-button" @click="logoutLastFM">Logout</button>
                 <span v-else style="color: #ff1100">No</span>
               </p>
             </div>
           </div>
+          <div v-if="lastFMEnabled" class="setting indented">
+            <YTMDSetting
+              v-model="scrobblePercent"
+              type="range"
+              name="Scrobble percent: "
+              description="Determines when a song is scrobbled"
+              min="50"
+              max="100"
```

Comment:
Worth checking that it works correctly at 100%.
As obviously the timer might last a little bit too long before it's then cancelled from the next song playing.

Reply:
What do you think about limiting this to `95` or subtracting a second from the timer: ` Math.round(state.videoDetails.durationSeconds * scrobblePercentDecimal) -1`

Reply:
I did find a bug. If the value for Last.FM progress is changed during a session, as the L.fm settings are cached in the extension, the progress value isn't updated until a Integration is turned off/on OR Program is restarted.

This would be ideal to update this value.